ui/components/multichain/account-list-item/account-list-item.js

```diff
@@ -239,35 +246,56 @@ export const AccountListItem = ({
             }}
           />
         ) : null}
+        {connectionStatus ? (
+          <Text
+            variant={TextVariant.bodyXs}
+            color={TextColor.successDefault}
+            data-testid="account-list-item-connected-status"
+          >
+            {connectionStatus}
+          </Text>
+        ) : null}
       </Box>
-      <ButtonIcon
-        ariaLabel={`${identity.name} ${t('options')}`}
-        iconName={IconName.MoreVertical}
-        size={IconSize.Sm}
-        ref={setAccountListItemMenuRef}
-        onClick={(e) => {
-          e.stopPropagation();
-          if (!accountOptionsMenuOpen) {
-            trackEvent({
-              event: MetaMetricsEventName.AccountDetailMenuOpened,
-              category: MetaMetricsEventCategory.Navigation,
-              properties: {
-                location: 'Account Options',
-              },
-            });
-          }
-          setAccountOptionsMenuOpen(!accountOptionsMenuOpen);
-        }}
-        data-testid="account-list-item-menu-button"
-      />
-      <AccountListItemMenu
-        anchorElement={accountListItemMenuElement}
-        identity={identity}
-        onClose={() => setAccountOptionsMenuOpen(false)}
-        isOpen={accountOptionsMenuOpen}
-        isRemovable={keyring?.type !== KeyringType.hdKeyTree}
-        closeMenu={closeMenu}
-      />
+      {showDisconnectIcon ? (
+        <>
+          <ButtonIcon
+            iconName={IconName.Slash}
+            size={IconSize.Sm}
+            data-testid="account-list-item-disconnect-buttonicon"
```

Comment:
At present this doesn't do anything -- should there be an onClick here?

Reply:
We are still tentative with the design on this, but I think the disconnect Icon will be removed completely, according to @kylefiedler  
[Figma](https://www.figma.com/file/LDlI7yjKBNIY60MtQv9QS0/Amon-Hen%2FV1?type=design&node-id=4878%3A841425&mode=design&t=uMbufVJ2quRetis8-1)